src/reference-manual/statements.qmd

```diff
@@ -365,42 +366,70 @@ To access accumulated log density up to the current execution point,
 the function `target()` may be used.
 
 
-## Sampling statements {#sampling-statements.section}
+## Distribution statements {#sampling-statements.section}
 
-Stan supports writing probability statements also in sampling
-notation, such as
+Stan supports writing probability statements also using distribution
+statements, for example
 
 ```stan
 y ~ normal(mu, sigma);
+mu ~ normal(0, 10);
+sigma ~ normal(0, 1);
 ```
+The symbol $\sim$ is called tilde. Due to historical reasons, the distribution statements used to be called "sampling statements" in Stan, but that term is not recommended anymore as it is less accurate description.
 
-The name "sampling statement" is meant to be suggestive, not
-interpreted literally.  Conceptually, the variable `y`, which may
-be an unknown parameter or known, modeled data, is being declared
-to have the distribution indicated by the right-hand side of the
-sampling statement. The above statement can read as "y is distributed
-as normal distribution with parameters mu and sigma.". The symbol `~`
-is called tilde.
+In general, we can read $\sim$ as "is distributed as," and overall this notation is used as a shorthand for defining distributions as
+$$
+\begin{aligned}
+   p(y| \mu, \sigma) & = \mathrm{normal}(y |  \mu, \sigma)\\
+   p(\mu) & = \mathrm{normal}(\mu |  0, 10)\\
+   p(\sigma) & = \mathrm{normal}^+(\sigma |  0, 1).
+\end{aligned}
+$$
+A collection of distribution statements define an unnormalized joint distribution as the product of component distributions
+$$
+p(y,\mu,\sigma) \propto p(y| \mu, \sigma )p(\mu) p(\sigma).
+$$
+In general, the product of arbitrary probability density functions is not a normalized probability density function---that is, it will be positive but will not in general integrate to 1---but the proportionality is sufficient for the Stan algorithms.
```

Comment:
I can't think of a situation where p(y, mu, sigma) won't integrate to 1 if the pieces integrate to 1.  Is there an easy counterexample?

Reply:
Uh, mixed joint and posterior here. Edited.